src/main/java/tests/OperationStrategyTest.java

```diff
@@ -0,0 +1,33 @@
+package tests;
+
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+
+import core.basesyntax.Operation;
+import core.basesyntax.OperationHandler;
+import core.basesyntax.OperationStrategy;
+import core.basesyntax.OperationStrategyImpl;
+import core.basesyntax.PurchaseOperation;
+import core.basesyntax.ReturnOperation;
+import core.basesyntax.SupplyOperation;
+import core.basesyntax.strategy.BalanceOperation;
+import java.util.HashMap;
+import java.util.Map;
+import org.testng.annotations.Test;
+
+public class OperationStrategyTest {
+    @Test
+    public void testOperationStrategy() {
+        Map<Operation, OperationHandler> operationHandlers = new HashMap<>();
```

Comment:
consider setting it up in beforeAll, and having a separate test for each handler